### PR TITLE
TASK: Add tests for reader objects

### PR DESCRIPTION
**Pull request by daniellienert**
Adds functional tests to Plurals, Number and DateReader

See #2116

### CODE REVIEW

**Review by kdambekalns — APPROVED** [review]

File: Neos.Flow/Tests/Functional/I18n/Cldr/Reader/PluralsReaderTest.php [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+ * This file is part of the Neos.Flow package.
+ *
+ * (c) Contributors of the Neos Project - www.neos.io
+ *
+ * This package is Open Source Software. For the full copyright and license
+ * information, please view the LICENSE file which was distributed with this
+ * source code.
+ */
+
+use Neos\Flow\I18n\Cldr\Reader\NumbersReader;
```
Comment (kdambekalns) [inline Neos.Flow/Tests/Functional/I18n/Cldr/Reader/PluralsReaderTest.php:16]: `PluralsReader`

File: Neos.Flow/Tests/Functional/I18n/Cldr/Reader/PluralsReaderTest.php [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    /**
+     * @var NumbersReader
+     */
+    protected $pluralsReader;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->pluralsReader = $this->objectManager->get(NumbersReader::class);
```
Comment (kdambekalns) [inline Neos.Flow/Tests/Functional/I18n/Cldr/Reader/PluralsReaderTest.php:32]: `PluralsReader`

File: Neos.Flow/Tests/Functional/I18n/Cldr/Reader/PluralsReaderTest.php [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    }
+
+    /**
+     * Data provider for returnsCorrectPluralForm
+     *
+     * @return array
+     */
+    public function quantities(): array
+    {
+        return [
```
Comment (kdambekalns) [inline Neos.Flow/Tests/Functional/I18n/Cldr/Reader/PluralsReaderTest.php:42]: All constants must be from `PluralsReader`

**Review by kdambekalns — COMMENTED** [review]
The test passed and I didn't look close enough. With #2118 applied, the test failed, and I noticed. Will push a fix to #2118…

**Comment by daniellienert** [comment]
@kdambekalns, gnarf, late night PRs :)

**Pull request merged**